core/templates/dev/head/pages/teach.html

```diff
@@ -0,0 +1,261 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
+  About

```

Comment:
Hi @charisseysabel -- re changing the title, I think this is what you want to modify!